plugins/vcenter_logs.yaml

```diff
@@ -0,0 +1,81 @@
+version: 0.0.1
+title: VMware vCenter
+description: Log parser for VMware vCenter
+parameters:
+  - name: listen_port
+    type: int
+    default: 5142
+  - name: listen_ip
+    type: string
+    default: "0.0.0.0"
+  - name: max_buffer_size
+    type: string
+    default: "1024kib"
+    required: false
+  - name: enable_tls
+    type: bool
+    default: false
+  - name: certificate_file
+    type: string
+    default: "/opt/cert"
+  - name: key_file
+    type: string
+    default: "/opt/key"
+
+
+  # Set Defaults
+  # {{$listen_address := default "0.0.0.0:5140" .listen_address}}
+  # {{$length := len $listen_address}}
+  # {{$listen_ip := default "0.0.0.0" .listen_ip}}
+  # {{$listen_port := default 5140 .listen_port}}
+  # {{$max_buffer_size := default "1024kib" .max_buffer_size}}
+  # {{$insecure := default false .insecure}}
+  # {{$certificate_file := default "" .certificate_file}}
+  # {{$key_file := default "" .key_file}}
+
+template: |
+  receivers:
+    tcplog:
+      listen_address: '{{ .listen_ip }}:{{ .listen_port }}'
+      max_log_size: {{ .max_buffer_size }}
+      attributes:
+        log_type: vmware_vcenter
+      add_attributes: true
+      {{ if .enable_tls }}
+      tls:
+        cert_file: {{ .certificate_file }}
+        key_file: {{ .key_file }}
+      {{ end }}
+      operators:
+        # vcenter will (sometimes) prepend an id to the messages, check
+        # for the id and drop it if it exsits
+        # example: '257 <14>1. . . '
+        - id: prefix_router
+          type: router
+          routes:
+            - output: pre_parser
+              expr: 'body matches "^\\d* "'
+          default: vcenter_parser
+
+        - id: pre_parser
+          type: regex_parser
+          regex: '^(?P<drop>\d* )(?P<syslog_message>[\w\W]*)'
+
+        - id: remove_drop
+          type: remove
+          field: body.drop
```

Comment:
Is this `body.drop`? I'm wondering if it should be `attributes.drop` instead based on regex parses usually parsing into `attributes`